{% extends "layout.html" %}
  
{% block title %}Model{% endblock %}
{% block lead %}Explain your model's assumptions, data, parameters, and results in a way that anyone could understand.{% endblock %}

{% block page_content %}


<div class="container-md my-3 position-relative">
  <div class="row">
    <aside class="d-none d-sm-block col-sm-3 pl-1">
        <nav id="navbar3" class="navbar navbar-light bg-light flex-column  align-items-stretch p-3 sticky-top">
          <div style="height: 50px"></div>
          <a class="navbar-brand" href="#">Model</a>
          <nav class="nav nav-pills flex-column">
              <a class="nav-link ms-3 my-1" href="#l1">Mechanism</a>
              <a class="nav-link ms-3 my-1" href="#l2">Model Assumptions</a>
              <a class="nav-link ms-3 my-1" href="#l3">Model Establishment</a>
              <a class="nav-link ms-3 my-1" href="#l4">Model Simulation</a>
              <a class="nav-link ms-3 my-1" href="#l4">Model Simulation</a>
          </nav>
        </nav>
    </aside>

    <div class=" col col-sm-9 mx-auto content-column">
      <div class="link_offset" id="l1"></div>
      <div class="pt-content-column-yellow p-4">
        <h1>Mechanism</h1>
        <hr>
        <br>
        <p>Oleic acid → acyl-CoA A → FadR protein departs from promoter PfadB → gene expresses normally, β-oxidation initiates.
        <p>Without oleic acid → FadR protein binds to promoter PfadB → gene expression is inhibited. </p>      </div>    
        <p>From the oleic acid induction mechanism, we can divide the expression of the FadR promoter into the following two scenarios:</p>  
        <img src=".\figures\2phenotype.png" alt="2phenotype" style="zoom:80%;" />  
        <p>The balance of oleic acid synthesis metabolism and decomposition metabolism in microorganisms is controlled by the transcriptional regulatory factor FadR. In the absence of oleic acid, it's a Growth phenotype with native negative autoregulation (NAR)  . At this time, fatty acid synthesis metabolism is activated. FadR binds to the promoter p-fadB of the bacterial β-oxidation pathway gene fadO, inhibiting the expression of fadO, thereby inhibiting bacterial β-oxidation. In the presence of excess oleic acid, it's a Production phenotype with engineered positive autoregulation (PAR) . Fatty acid degradation metabolism is activated. After FadR couples with oleic acid, it departs from the promoter p-fadB of fadO, allowing fadO expression, and bacterial β-oxidation is completed.</p>
      
        <div class="link_offset" id="l2"></div>
        <div class="pt-content-column-yellow p-4">
          <h1>Model Assumptions</h1>
          <hr>
          <br>
          <p>To establish a more reasonable mathematical model, the following assumptions are made:<br>
            1. The activation state of the oleic acid inducer (corresponding to the presence/absence of oleic acid environment) can be determined by the concentration of the regulator FadR. The expression of the product synthesis enzyme and the synthesis of the product itself depend on the control factor FadR. Therefore, we only define the state of the oleic acid inducer by the concentration of FadR.<br>
            2. FadD expression is solely controlled by FadR. In native E. coli, the  expression of FadD under aerobic conditions is co-regulated by FadR and  the CRP-cAMP complex (EcoCyc). However, in the presence of common carbon sources like glucose, the expression level of CRP-cAMP is very low, so  its effect can be neglected.<br>
            3. The expression rate of the enzyme in two oleic acid inducer states can be modeled in the form of a Hill function. Studies have shown that the expression of gene products takes the form of a Hill function [4], either as $P_T(T)=\frac{a \cdot(K \cdot T)^n}{1+(K \cdot T)^n}$ for $T$-activated expression, or as $P_T(T)=\frac{a}{1+(K \cdot T)^n}$ for $T$-inhibited expression, where $a$ is the maximum expression rate, $K$ is the affinity with which transcription factor $T$ binds to the operator and controls expression, and $n$ is the Hill coefficient.<br>
            4. The growth rate of engineered bacteria linearly depends on the growth-related enzyme Eg. According to research by Usui et al. [3], the cell growth rate decreases in an approximately linear fashion. We capture this phenomenon and define the growth rate of the cell using a linear function as follows: $\lambda\left(E_g\right)=E_g \cdot \lambda_{\min }+\left(\lambda_{\max }-\lambda_{\min }\right)$. Where $\lambda_{\min }$ and $\lambda_{\max }$ represent the growth rates at zero and maximum expression of $E_g$, respectively.<br>
            5. Enzymes and transcription factors do not undergo active degradation. To our knowledge, proteins in the system we study are not actively  degraded. Over the duration of cell doubling, protein dilution through  cell growth and division is the main way proteins are lost from the  system, so we model the decay of protein $E$ as growth dilution $\lambda \cdot E$.
            </p>
        </div>  

          <div class="link_offset" id="l3"></div>
          <div class="pt-content-column-yellow p-4">
            <h1>Model Building</h1>
            <hr>
            <br>
            <p>A model which represents multi-component and time-varying dynamic system is widely used in various biological problems. Among all of the models, differential equation is the top of the list. To illustrate it, the regulatory network can be represented by an array of ordinary differential equations, where the interaction between a multitude of molecules (such as mRNA or protein) are quantified by the law of mass action[1]. These equations designate the level of each protein or mRNA as a function of other components in the development of the system. These models usually include time-related variables, such as the concentration of protein and mRNA, other parameters, such as protein degradation parameters, are also included. </p>
           <h2>Reaction mechanism</h2>
           <p>We used oxygen, pH, and lactate as unique indicators of colorectal cancer tumor tissue to construct bacterial biosensing systems capable of distinguishing unique organ environments. To represent the behavior of the biosensor, we used ordinary differential equations to describe the production rate of the red fluorescent protein (RFP) controlled by different biosensor modulators, including activators for the hypoxia and pH biosensors and repressors for the lactate biosensors.</p>
           <p><b>For hypoxia biosensors</b>, we used a hypoxia-sensing promoter (pPepT) that is primarily regulated by the transcriptional activator, fumarate and nitrate reduction regulatory protein (FNR) In the absence of oxygen, FNR generates a transcriptionally active homodimer. However, the cluster is degraded in the presence of oxygen, which dissociates the FNR dimer into inactive monomers. (fig 1)</p>
           <p>Thus, by measuring red fluorescent protein (RFP) expressed under the control of the pPepT on a plasmid, we detected elevated levels of fluorescence in response to hypoxic conditions.</p>
           <img src="https://static.igem.wiki/teams/4156/wiki/model/fig1.png" class="d-block mx-auto mt-3" width="50%">
        <center>
            <font size='2'>fig1[2]</font>
        </center>    
           <p><b>For lactate biosensor</b>, We derive it from the native lldPRD operon, to detect lactic acid fermentation by host mammalian cells[3]. This lactate-sensing system was constructed on two plasmids: 1. a lactate-inducible reporter plasmid which drives expression of a gene of interest. 2. a repressor plasmid, which produces the repressor LldR that dimerizes to inhibit expression of the reporter gene unless bound to lactate. (fig2)</p>
           <p>Hence, in response to increasing concentrations of lactate in the culture medium, we observed a corresponding increase in RFP.</p>
           <img src="https://static.igem.wiki/teams/4156/wiki/model/fig2.png" class="d-block mx-auto mt-3" width="50%">
        <center>
            <font size='2'>fig2</font>
        </center>    
           <p><b>For pH biosensor</b>, we engineered the pH-sensitive promoter pCadC among other systems[4] that is regulated by a membrane-tethered activator protein (CadC), which shows increased activity in acidic medium compared with medium at a neutral pH.</p>
           <p>Therefore, comparing with medium at a neutral pH, we observed a relative high level of fluorescence of RFP in the acidic medium.(fig3)</p>
           <img src="https://static.igem.wiki/teams/4156/wiki/model/fig3.png" class="d-block mx-auto mt-3" width="50%">
        <center>
            <font size='2'>fig3</font>
        </center>    
           <p>Based on the above reaction mechanism, we will first write the dynamics of the whole process and then build a mathematical modeling to both gain insight into and make predictions about their behavior. </p>
           <h2>The dynamics of the promoters</h2>
           <p>The dynamics of the promoters are given by the following set of reactions:</p>
           $$
           \begin{equation}
	\begin{aligned}
	& FNR_2 \underset{k_{-O_2}}{\stackrel{k_{O_2}}{\rightleftharpoons}} 2FNR\\
	& pPepT+FNR_2 \stackrel{k_{p1}}{\longrightarrow} pPepT^+\\
	& LldR_2 \underset{k_{-L}}{\stackrel{k_{L}}{\rightleftharpoons}} 2LldR\\
	& pLldR+LldR_2 \stackrel{k_{l1}}{\longrightarrow} pLldR^-\\
	& pCadC+CadC \stackrel{k_{c1}}{\longrightarrow} pCadC^+\\
	\end{aligned}
\end{equation}
           $$
           <p>Where: $ pPepT^+ $ represent the states of pPepT bound with activator.
            $ pLldR^- $ represent the states of pLldR bound with repressor.<br>
            $ pCadC^+ $ represent the states of pCadC bound with activator.<br>
            $ FNR $ represents FNR monomer.<br>
            $ FNR_2 $ represents FNR dimer, which is the activator of $ pPepT $.<br>
            $ LldR $ represents LldR monomer.<br>
            $ LldR_2 $ represents LldR dimer, which is the repressor of $ pLldR $.<br>
            $ CadC $ represents CadC monomer.<br>
            $ CadC_2 $ represents CadC dimer, which is the activator of $ pCadC $.
            </p>
            <p>That is the dynamics of the promoters.</p>
            <h2>The dynamics of the transcription, translation</h2>
            <p>Then we move to the set of reactions govern the transcription, translation of each gene/protein: </p>
            $$
            \begin{equation}
	\begin{aligned}
	pPepT^+ & \stackrel{k_{p2}}{\longrightarrow} pPepT^+ + m_{O_2}\\
	pPepT & \stackrel{k_{p2'}}{\longrightarrow} pPepT + m_{O_2}\\
	m_{O_2} + & \stackrel{k_{p3}}{\longrightarrow} m_{O_2}+RFP\\
	pLldR & \stackrel{k_{l2}}{\longrightarrow} pLldR + m_{L}\\
	m_{L} & \stackrel{k_{l3}}{\longrightarrow} m_{L}+RFP\\
	pCadC^+ & \stackrel{k_{c2}}{\longrightarrow} pCadC^+ + m_{pH}\\
	pCadC & \stackrel{k_{c2'}}{\longrightarrow} pCadC + m_{pH}\\
	m_{pH} & \stackrel{k_{c3}}{\longrightarrow} m_{pH}+RFP\\
	\end{aligned}
\end{equation}
            $$
            <p>where $m_{O_2}$ represents the mRNA which is transcribed of gene segment with pPepT as its promoter;</p>
            <p>$m_{L}$ represents the mRNA which is transcribed of gene segment with pPepT as its promoter;</p>
              <p>$m_{pH}$ represents the mRNA which is transcribed of gene segment with pPepT as its promoter;</p>
              <p>$ RFP $ represents the translation product: fumarate and nitrate reduction regulatory protein(RFP);
              </p>
              <h2>The dynamics of the decay</h2>
              <p>Finally, the mRNA, protein, and the promoter which is bound with activator/repressor decay exponentially, and all tagged forms of decay enzymatically according to the following set of reactions:</p>
              $$
              \begin{equation}
	\begin{aligned}
	RFP &\stackrel{\gamma}{\longrightarrow} \varnothing\\
	m_{O_2} &\stackrel{\gamma_{p4}}{\longrightarrow} \varnothing\\
	m_{L} &\stackrel{\gamma_{l4}}{\longrightarrow} \varnothing\\
	m_{pH} &\stackrel{\gamma_{c4}}{\longrightarrow} \varnothing\\
	pPepT^+ &\stackrel{\gamma_{p5}}{\longrightarrow} pPepT\\
	pLldR^- &\stackrel{\gamma_{l5}}{\longrightarrow} pLldR\\
	pCadC^+ &\stackrel{\gamma_{c5}}{\longrightarrow} pCadC\\
	\end{aligned}
\end{equation}

              $$
              <h2>ODE model I: Expression of RFP gene</h2>
              <p>According to the above reaction mechanism, this reactions rates can be expressed using the Law of Mass Action. We use [x] to denote the concentration of substance x. Therefore, constitutive expression of RFP is mathematically described by an ordinary differential equations (ODE) that captures the species dynamics.</p>
              <p>Below is the reaction dynamics of the hypoxia biosensor</p>
              $$
              \begin{equation}
		\left\{
		\begin{aligned}
		&\frac{d\left[FNR_2\right]}{d t}=k_{-O_2}[FNR]^2-k_{p_1}[pPepT]\left[FNR_2\right] \\
		&\frac{d\left[FNR\right]}{d t}=k_{O_2}\left[FNR_2\right] \\
		&\frac{d\left[pPepT\right]}{d t}=-k_{p_1}[pPepT]\left[FNR_2\right]+\gamma_{p_5}\left[pPepT^{+}\right] \\
		&\frac{d\left[pPepT^{+}\right]}{d t}=k_{p_1}[pPepT]\left[FNR_2\right]-\gamma_{p_5}\left[pPepT^{+}\right] \\
		&\frac{d\left[m_{O_2}\right]}{d t}=k_{p_2}\left[pPepT^{+}\right]+k_{p'_2}\left[pPepT^{+}\right]-\gamma_{p_4}\left[m_{O_2}\right] \\
		&\frac{d[R F P]}{d t}=k_{p 3}\left[m_{O_2}\right]-\gamma[R F P]
		\end{aligned}
		\right.
	\end{equation}

              $$
              <p>Next is the reaction dynamics of the lactic acid biosensor</p>
              $$
              \begin{equation}
              \left\{
              \begin{aligned}
              &\frac{d\left[LldR_2\right]}{d t}=k_{-L}[LldR]^2-k_{l_1}[pLldR]\left[LldR_2\right] \\
              &\frac{d\left[LldR\right]}{d t}=k_{L}\left[LldR_2\right] \\
              &\frac{d\left[pLldR\right]}{d t}=-k_{l_1}[pLldR]\left[LldR_2\right]+\gamma_{l_5}\left[pLldR^{-}\right] \\
              &\frac{d\left[pLldR^{-}\right]}{d t}=k_{l_1}[pLldR]\left[LldR_2\right]-\gamma_{l_5}\left[pLldR^{-}\right] \\
              &\frac{d\left[m_L\right]}{d t}=k_{l_2}\left[pLldR^{-}\right]-\gamma_{l_4}\left[m_L\right] \\
              &\frac{d[RFP]}{dt}=k_{L3}\left[m_L\right]-\gamma[RFP]
              \end{aligned}
              \right.
            \end{equation}
          
              $$
            <p>Last but not least, this is the reaction dynamics of the lactic pH biosensor</p>
            $$
            \begin{equation}
		\left\{
		\begin{aligned}
		&\frac{d\left[pCadC\right]}{d t}=-k_{c_1}[pCadC]\left[CadC\right]+\gamma_{c_5}\left[pCadC^{+}\right] \\
		&\frac{d\left[pCadC^{+}\right]}{d t}=k_{c_1}[pCadC]\left[CadC\right]-\gamma_{c_5}\left[pCadC^{+}\right] \\
		&\frac{d\left[m_{pH}\right]}{d t}=k_{c_2}\left[pCadC^{+}\right]+k_{c'_2}\left[pCadC^{+}\right]-\gamma_{c_4}\left[m_{pH}\right] \\
		&\frac{d[R F P]}{d t}=k_{c 3}\left[m_{pH}\right]-\gamma[R F P]
		\end{aligned}
		\right.
	\end{equation}

            $$
            <h3>Simplification of Model</h3>
            <p>Because the change of RFP concentration over time is our most concerned index. However, the above equations of ordinary differential equations require a large amount of computation and a large number of solving parameters, and the time cost of complete simulation is too high. Therefore, we can simplify the above equations and only focused on the relationship between RFP concentration and time to reduce the computational cost [5].</p>
            <p>Considering a gene regulated by a single regulator, this transcriptional interaction is described as “transcription factor X regulates gene Y”. In the absence of input signals, X is inactive and no Y is generated. Upon the presence of an external signal, the transcription factor rapidly changes to the active form and binds to the promoter of gene Y. Gene Y starts to be transcribed and mRNA is translated, leading to the accumulation of proteins. The concentration change of Y is described by the dynamic equation:</p>
            $$
            \begin{equation}
\frac{dy}{dt}=\beta-\gamma y.
\end{equation}
            $$
            <p>Here, $ \beta $ is the rate at which the gene produces protein Y, and $ \alpha $ is the sum of the degradation rate and dilution rate of the protein. y is the concentration of protein Y. The difference between production and degradation/dilution results in a change in the concentration of Y.</p>
            <p>In some of the more accurate models, $ \beta $ would vary with the concentration of transcription factor change. This can be described by the strength of the effect of a transcription factor on the transcription rate of its target gene. </p>
            <p>Let us consider first the production rate of protein Y controlled by a single transcription factor X. The number of molecules of protein Y produced per unit time $ f(x^*) $ is a function of the concentration of X in its active form $ X^*$. Then, the input functions of many real genes can be expressed in the form of Hill functions [6]:</p>
            $$
            \begin{equation}
	f(x^*) =
	\begin{cases}
		& \frac{\beta}{1+(\frac{K}{x^*})^{n}},\quad (X^* \text{ is an activator)}\\
		& \frac{\beta}{1+(\frac{x^*}{K})^{n}},\quad (X^* \text{ is an repressor)}\\
	\end{cases}
\end{equation}

            $$
            <p>Where, $ \beta $ is the maximum expression level of the promoter;<br>
              $ n $ controls the steepness of the function;<br>
              $ x^* $ is the concentration of X in its active form $ X^*$;<br>
              $ K $ defines the concentration of active transcription factor, and it is related to factors such as the chemical affinity between a transcription factor and its site on the promoter.
              </p>
              <p>Combining with above equations, we can get the dynamic equation that describes the concentration change of Y, namely:</p>
              $$
              \begin{equation}
	\frac{dy}{dt} =
	\begin{cases}
		& \frac{\beta}{1+(\frac{K}{ x^*})^{n}}-\gamma y,\quad (X^* \text{ is an activator)}\\
		& \frac{\beta}{1+(\frac{ x^*}{K})^{n}}-\gamma y,\quad (X^* \text{ is an repressor)}\\
	\end{cases}
\end{equation}

              $$
              <p>Since transcription factor X requires an environmental signal to transition to the active state, in order to describe this process, we denote $x_T$ as the total transcription factor X concentration, $x$ as the transcription factor concentration X without binding environmental signal, and $x^*$ as the transcription factor concentration after binding environmental signal (namely active mode, $X^*$). S is the concentration of environmental signal in the environment. </p>
              <p>From the law of conservation of mass, we have:</p>
              $$\begin{equation} x_T = x^* + x \end{equation}$$
              <p>
                Let $k_{on}$ denote the binding rate of the environmental signal and the transcription factor, and $k_{on}$ denote the separation rate of the environmental signal and the transcription factor. We have:
                </p>
                $$\begin{equation} \frac{dx^*}{dt}= k_{on} xS - k_{off}x^*\end{equation}$$
                <p>In general, $k_{on}>>k_{off}$, hence, the reaction will quickly achieve chemical equilibrium relation, at this point, $\frac{dx^*}{dt}=0$.[7] We can get: </p>
                $$\begin{equation} x^* = \frac{x_TS}{S+k_s}\end{equation}$$
                <p>where, $k_s=\frac{k_{off}}{k_{on}}$ is the equilibrium constant of the signal-transcription factor binding reaction. From $ x^*=x_T-x $, we have: </p>
                $$\begin{equation} x = \frac{x_Tk_s}{S+k_s}\end{equation}$$
                <p>Based on the above analysis, we developed a simplified system of ordinary differential equations that describe the dynamic expression of RFP(y) in response to varying levels of environmental lactate concentration (L; equation (1)), oxygen concentration (O2; equation (2)) and pH (H^+ concentration) (H+; equation (3)), regulated by transcriptional factors LldR, FNR and CadC, respectively.</p>
                $$
                \begin{equation} 
	\begin{aligned} 
		\left\{ 
			\begin{array}{lc} 
				\frac{dy}{dt}=\frac{\alpha_y}{1+\frac{K_1}{1+K_{L}L}}-\gamma_{y}y,\\ 
				\frac{dy}{dt}=\frac{\alpha_y}{1+K_2(1+K_{O_{2}}O_{2})}-\gamma_{y}y,\\ 
				\frac{dy}{dt}=\frac{\alpha_y}{(1+K_3)H^{+}+K_{H}}-\gamma_{y}y,\\ 
				y(0) = 0
			\end{array}
		\right.
	\end{aligned} 
\end{equation}

$$
            <p>where: $\alpha_y$ is maximum transcription rate/maximum production rate;<br>
              $\gamma_y$ is total degradation/dilution rate of RFP; <br>
              $ K_1 $ is Binding affinity of transcription factor LIdR to promoter plldP;<br>
              $ K_L $ is Binding affinity of transcription factor LIdR to lactic acid;<br>
              $ K_2 $ is Binding affinity of transcription factor FNR to promoter pTeT;<br>
              $ K_{O_2} $ is affinity of transcription factor FNR for oxygen binding;<br>
              $ K_3 $ is Binding affinity of transcription factor CadC to promoter pCadC;<br>
              $ K_H $ is affinity of the transcription factor CadC for hydrogen ion binding.
              </p>
              <p>Then, We use this equation to describe the dynamic expression of RFP(y) in different biosensors.</p>
              <h2>ODE model II: Expression of lysis gene</h2>
              <p>In this model, a new lysis gene was added to the gene fragment of the biosensor. Therefore, we hope to establish a model to describe and predict the changes of biosensor population with time after the addition of lysis genes.</p>
              <p>We denote $r$ as the growth rate of CR flora, $N$ as the number of CR flora. If we assume that $r$ doesn't change over time, then the simplest model is that:</p>
              $$\begin{equation}\frac{dN}{dt} = rN\end{equation}$$
              <p>In practice, for the limit of nutrients and living space in the bacterial culture environment, there will be a maximum upper limit number of flora $N_{max}$, and at the same time, as the number of flora increases, the growth rate of the flora should decrease, so $r$ is related to $N$. Under the assumption that the natural growth rate is $r_0$ we can obtain:</p>
              $$\begin{equation}r(N) = aN+r_0\end{equation}$$
              <p>When $ N = N_{max} $, namely, the number of flora reaches its maximum, at which point the growth rate of the flora should be 0, so it can be obtained:</p>
              $$\begin{equation}r(N_{max})=aN_{max}+r_0=0\end{equation}$$
              <p>Therefore, we have:</p>
              $$\begin{equation}a=-\frac{r_0}{N_{max}}\end{equation}$$
              <p>Thus, a logistics equation that describe the growth of the flora can be obtained[8]:</p>
              $$\begin{equation}\frac{dN}{dt}=r_0\left(1-\frac{N}{N_{max}}\right)N\end{equation}$$
              <p>The above model is a description and prediction of the number of bacterial populations in natural environments over time. However, this equation doesn’t apply to the biosensor population in this experiment . Therefore, we try to modify this equation.</p>
              <p>The time-dependent relationship of the translation product produced by the biosensor is shown in the formula 10. Therefore, we can obtain the following model for the lactic acid biosensor with lysis gene:</p>
              $$
              \begin{equation}
	\begin{cases}
		\displaystyle \frac{dN}{dt} &= r_0\left(1-\frac{N}{N_{max}}\right)N-\frac{\alpha_y}{1+\frac{K_1}{1+K_{L}L}}N\\
		N(0) &= N_0\\
	\end{cases}
\end{equation}
$$
<p>Similarly, we can obtain mathematical models describing the pH biosensor and hypoxia biosensor with lysis genes:</p>
$$
\begin{equation}
	\begin{cases}
		\displaystyle \frac{dN}{dt} &= r_0\left(1-\frac{N}{N_{max}}\right)N-\frac{\alpha_y}{(1+K_3)H^{+}+K_{H}}N\\
		N(0) &= N_0\\
	\end{cases}
\end{equation}

$$
$$
\begin{equation}
    \begin{cases}
        \displaystyle \frac{dN}{dt} &= r_0\left(1-\frac{N}{N_{max}}\right)N-\frac{\alpha_y}{1+K_2(1+K_{O_{2}}O_{2})}N\\
        N(0) &= N_0\\
    \end{cases}
\end{equation}


$$
              <p>Then, we use this equation to describe the dynamic of number in different biosensors.</p>
          </div>      
          
  
            <div class="link_offset" id="l4"></div>
            <div class="pt-content-column-yellow p-4">
              <h1>Model Simulation</h1>
              <hr>
              <br>
              <h2>Determined parameter values</h2>
              <h3>Model I: Expression of RFP gene </h3>
              <p>For the above biosensors, we first constructed lactic acid biosensors with pLldR as their promoters, and detected curves of RFP concentration over time under the experimental conditions of lactic acid concentration of 0Mm, 0.5mm, 1mM, and 1.5mm, respectively. Three groups of parallel experiments were performed for each concentration gradient. </p>
              <p>For lactate biosensor, the model is as follows:</p>
              $$
              \begin{equation}
	\frac{dy}{dt} = \frac{\alpha_y}{1+\left(\frac{K_1}{1+K_LL}\right)}-\gamma_y y, \quad y(0) = 0
\end{equation}
$$  
              <p>If we denote $y_j^i $ as the experimental data of the $ i $ th parallel group at $ j $th sampling time, and $ N  $is the number of data samples in each parallel group, then the data sampling time can be denoted as $\{t_j\}_{j=1}^N $. Since our experiment was conducted in triplicate control group, $i \in\{1,2,3\}$,.</p>
              <p>Our goal is to select the above parameters reasonably so as to minimize the error between the curve of the formula 14 and the experimental data.  Geometrically, our model curve needs to be the closest to the experimental data curve, mathematically, we need to find the parameter $\alpha_y^*, K_1^*, K_L^*, \gamma_y^* $that minimizes the error between the model and the experimental data and satisfies the following conditions:	</p>
              $$
              \begin{equation} \min \sum_{j=1}^{N} \left(\tilde{y_j}-\bar{y_j}\right)^2, \quad \bar{y_j} = \frac{\sum_{i=1}^3y_j^i}{3}\end{equation}
              $$
              $$\begin{equation} \left\{ \begin{aligned} 
    \displaystyle \frac{dy}{dt} &= \displaystyle \frac{\alpha_y^*}{1+\left(\frac{K_1^*}{1+K_L^*L}\right)}-\gamma_y^* y \\
    \tilde{y_j} &= y(t_j) \\ 
\end{aligned} \right. \end{equation} 
$$
<p>The objective function is in fact called least square method (LSM). The essence of this problem is an optimization problem. So, for the target function: $ \sum_{j=1}^{N} \left(\tilde{y_j}-\bar{y_j}\right)^2 $, we apply the particle swarm optimization algorithm(PSO)[9],To find the global optimal parameter $\alpha_y^*, K_1^*, K_L^*, \gamma_y^* $, the following is the iterative solution procedure of the particle swarm optimization algorithm.</p>
<p>Its basic algorithm is as follows: Let $S$ be the number of particles in the swarm, each having a position $x_i \in R^n$ in the search-space and a velocity $v_i \in R^n$. Let $p_i$ be the best known position of particle $ i $ and let $ g $ be the best known position of the entire swarm. A basic PSO algorithm is then:</p>
<center>
<img src="https://static.igem.wiki/teams/4156/wiki/model/fakecode.png" class="d-block mx-auto mt-3" width="80%"></center>
<p>The values $b_{lo}$ and $b_{up}$ represent the lower and upper boundaries of the search-space respectively. The $w$ parameter is the inertia weight. The parameters $\phi_p$ and $\phi_g$ are often called cognitive coefficient and social coefficient. The termination criterion can be the number of iterations performed, or a solution where the adequate objective function value is found. The parameters $w$, $\phi_p$, and $\phi_g$ are selected by the practitioner and control the behavior and efficacy of the PSO method.</p>
<center><iframe title="LZU-CHINA: Model Construction (2022) [English]" width="85%" height="500px" src="https://video.igem.org/videos/embed/de40a8a0-1bee-4115-89f0-c3b25a30c0fb" frameborder="0" allowfullscreen="" sandbox="allow-same-origin allow-scripts allow-popups"></iframe></center>
<p>From the above animation, we can see that after nearly 70 iterations, our algorithm finally finds the parameters $\alpha_y^*, K_1^*, K_L^*, \gamma_y^* $ that minimize the error between the model and the experimental data, which is the curve of the model that is closest to the experimental data curve.</p>
<p>After that, we selected the average RFP concentration of three parallel groups as the data point, plotted the test data curve that generated the change of RFP concentration over time, and plotted the model curve and the test data curve at the same time, as shown in the following figure:</p><img src="https://static.igem.wiki/teams/4156/wiki/model/plldr-times-curve.png" class="d-block mx-auto mt-3" width="95%">  
<p>Then, we constructed pH biosensors with pCadC as their promoters, and detected curves of RFP concentration over time under the experimental conditions of  $H^+$ concentration in 10^{-5.3}, 10^{-5.8}, 10^{-6.3}, 10^{-7.3} respectively. Three groups of parallel experiments were performed for each concentration gradient. </p>

<p>Similarly, we use the above method and PSO to find best parameters and plot time curves of the models.</p>
<img src="https://static.igem.wiki/teams/4156/wiki/model/pcadc-times-curve.png" class="d-block mx-auto mt-3" width="95%">  
<p>Finally, we constructed hypoxia biosensors with pPepT as their promoter, and detected curves of RFP concentration over time under the experimental conditions of $ O_2 $ volume fraction of 0%, 20%, respectively. Three groups of parallel experiments were performed for each volume fraction gradient. </p>
<p>Similarly, we use the above method and PSO to find best parameters and plot time curves of the models.</p>
<img src="https://static.igem.wiki/teams/4156/wiki/model/ppept-times-curve.png" class="d-block mx-auto mt-3" width="95%">  
<h3>Model II: Expression of lysis gene</h3>
<p>For the above biosensors, we first constructed lactic acid biosensors with pLldR as their promoters, and connected the red fluorescent protein gene and the φ174E lysis gene, then detected curves of $OD_{600}$ over time under the experimental conditions of lactic acid concentration in 0Mm, 0.5mm, 1mM, and 1.5mm, respectively. </p>
<p>Three groups of parallel experiments were performed for each concentration gradient. </p>
<p>For lactate biosensor, the model is as follows:</p>
$$
\begin{equation}
	\begin{cases}
		\displaystyle \frac{dN}{dt} &= r_0\left(1-\frac{N}{N_{max}}\right)N-\frac{\alpha_y}{1+\frac{K_1}{1+K_{L}L}}N\\
		N(0) &= N_0\\
	\end{cases}
\end{equation}
$$
<p>Similarly, we still use PSO and LSM to global optimization to get the optimal parameters, then, the obtained optimal model and the experimental data points after the average of the three experiments were plotted in the same coordinate system, and the curves were shown in the figure below.</p>
<img src="https://static.igem.wiki/teams/4156/wiki/model/plldr-n-times-curve.png" class="d-block mx-auto mt-3" width="95%">  
              <p>In the figure above, the circles with different colors represent the data points obtained by experimental groups with different lactic acid concentrations, the dashed lines represent the curves of experimental groups with different lactic acid concentrations, and the solid lines with different colors represent the curves of the obtained model.</p>
              <p>Then for pH biosensors with pCadC as their promoters, then we detected curves of $OD_{600}$ over time under the experimental conditions of $H^+$ concentration in 10^{-5.3}, 10^{-5.8}, 10^{-6.3}, 10^{-7.3} respectively. Three groups of parallel experiments were performed for each concentration gradient. </p>
              <img src="https://static.igem.wiki/teams/4156/wiki/model/pcadc-n-times-curve.png" class="d-block mx-auto mt-3" width="95%">  
              <p>Finally, we constructed hypoxia biosensors with pPepT as their promoter, and detected curves of $OD_{600}$ over time under the experimental conditions of $ O_2 $ volume fraction of 0%, 20%, respectively. Three groups of parallel experiments were performed for each volume fraction gradient. </p>
              <img src="https://static.igem.wiki/teams/4156/wiki/model/ppept-n-times-curve.png" class="d-block mx-auto mt-3" width="95%">  
              <p>The estimation of parameters can be found in the Appendix A. Parameters, models are built by MATLAB as their tool. Codes of these models can be found in Appendix B. Codes.</p>
              <iframe id="myframe" src="https://static.igem.wiki/teams/4156/wiki/model/appendix-a-parameters.pdf"
          width="100%" height="600" frameborder="0"></iframe>
          <br>
          <iframe id="myframe" src="https://static.igem.wiki/teams/4156/wiki/model/appendix-b-codes.pdf"
          width="100%" height="600" frameborder="0"></iframe>
          <p> </p>
              <h2>Sensitivity analysis</h2>
              <p>In this part, we analyze the sensitivity of the model I. Since the deviation between the model obtained by the lactate biosensor with the lactate concentration of 0.5mM and the actual data is the smallest, we use the model corresponding to the lactate biosensor with the lactate concentration of L=0.5mM for analysis.
              <p>According to Appendix A. Parameters, the optimal model Parameters obtained by us are:</p>
              <center><table class="mt">
                <tr>
                  <td>Parameter</td>
                  <td>$\alpha_y^*$</td>
                  <td>$K_1^*$</td>
                  <td>$K_L^*$ </td>
                  <td>$\alpha_y^*$</td>
                </tr>
                <tr>
                <td>Optimized Value</td>
                <td>9585.988858886</td>
                <td>2919.519950676</td> 
                <td> 0.1000000000000</td>
                <td> 0.1000000000000</td>
                </tr>
              </table></center>
            
              <p>Therefore, we first try to see how the model changes when $\alpha_y$varies in the range of $[90\%\alpha_y^*,110\%\alpha_y^*]$.</p>
              <p>Firstly, we devide $[ 90 \ % \ alpha_y ^ *, 110 \ % \ alpha_y ^ *] $ in 20 parts uniformly, to give out 20 values of $\ alpha_y ^i $, where $i\ in\ {1 ,\cdots, 20 }$.  Then we plot time curves of these differential equations.</p>
              $$\begin{equation}
              \frac{dy^i}{dt}=\frac{\alpha_y^i}{1+\frac{K_1^*}{1+K_{L}^*L}}-\gamma_{y}^*y^i, \quad i=1,\dots,20\\
            \end{equation}
            $$
              <p>After that, we plotted the resulting function curve in a unified coordinate system, with time as the X-axis, RFP concentration as the Y-axis, and $\alpha_y^ i $value as the z-axis, as shown below:</p>
              <img src="https://static.igem.wiki/teams/4156/wiki/model/alpha-y.png" class="d-block mx-auto mt-3" width="95%">  
              <p>Below is a plot of the relationship between the experimental data points and the changing function curve. The different colors of the data points indicate the different number of groups in the experimental group when they were collected. The black solid line represents the best parameter curve obtained by PSO, and the different colored dotted lines represent the curve after the deviation of the parameters</p>
              <img src="https://static.igem.wiki/teams/4156/wiki/model/alpha-yy.png" class="d-block mx-auto mt-3" width="95%">  
              <p>After that, we select 200 time points $t_j$ equally spaced, where $j\in\{1,\dots,200\}$, to calculate the maximum deviation ratio $\eta_j$ of the curve corresponding to $t_j$ from the optimal curve at different times for different values of $\alpha_y^ i $, which can be calculated as follows:</p>
              $$\begin{equation}
              \eta_j = \max_{i\in\{i,\dots,20\}}\left(\frac{y^i(t_j)-y(t_j)}{y(t_j)}\right)^2 \times 100\%
            \end{equation}
          $$
          <p>Finally, we plot $(t_j,\eta_j)$ with smooth curves to obtain the curve of the maximum deviation ratio curve over time.</p>
          <img src="https://static.igem.wiki/teams/4156/wiki/model/the-percentage-of-deviation-1.png" class="d-block mx-auto mt-3" width="95%">  
          <p>As can be seen in the figure above, the maximum deviation ratio $\eta_j$ of our curve increases monotonically with time. The maximum deviation ratio of $\eta_j$ remains within 1% even when the parameters change within 10%, which shows that our model has high stability.</p>
          <p>We also carry out a similar sensitivity analysis for $K_1^*$, $K_{L}^*$, $\gamma_{y}^*$ at the same time, the variation range of the curve still remains in the range of $[90\%,110\%]$, therefore, we can draw the differential equation curve as shown in the following figure.</p>
          <img src="https://static.igem.wiki/teams/4156/wiki/model/k-1.png" class="d-block mx-auto mt-3" width="95%">  
          <img src="https://static.igem.wiki/teams/4156/wiki/model/k-l.png" class="d-block mx-auto mt-3" width="95%"> 
          <img src="https://static.igem.wiki/teams/4156/wiki/model/gamma-y.png" class="d-block mx-auto mt-3" width="95%"> 
          <p>Finally, we use $\ alpha_y ^ * $, $K_1 ^ * $, $K_ {L} ^ * $, $\ gamma_ { y ^ *} $ respectively to draw the corresponding curve of optimal curve of maximum deviation and plot these under the same axis, the result is as follows:</p>
          <img src="https://static.igem.wiki/teams/4156/wiki/model/the-percentage-of-deviation.png" class="d-block mx-auto mt-3" width="95%"> 
          <p>From the above figure, we can see that the maximum deviation ratio of our curve $\eta_j$ remains within 1.5% even when the parameters are varied by 10%, indicating that our model is stable.</p>
          <p>Similarly, sensitivity analysis was also carried out for other concentrations of lactic acid biosensors and two other types of biosensors, which again proved that our model <b>had high stability</b>.</p>
            </div> 


              <div class="link_offset"></div>
              <div class="pt-content-column-yellow p-4">
                <h1>Reference</h1>
                <hr>
                <br>
                <p>[1] Guldberg, C.M.; Waage, P. (1879). "Ueber die chemische Affinität" [On chemical affinity]. Journal für praktische Chemie. 2nd series (in German). 19: 69–114. doi:10.1002/prac.18790190111. Reprinted, with comments by Abegg in Ostwalds Klassiker der Exacten Wissenschaften, no. 104, Wilhelm Engleman, Leipzig, 1899, pp 126-171<br>
                  [2] Chien, Tiffany, et al. "Enhancing the tropism of bacteria via genetically programmed biosensors." Nature biomedical engineering 6.1 (2022): 94-104.<br>
                  [3] Weghoff, M. C., Bertsch, J. & Muller, V. A novel mode of lactate metabolism in strictly anaerobic bacteria. Environ. Microbiol. 17, 670–677 (2015).<br>
                  [4] Stirling, F. et al. Synthetic cassettes for pH-mediated sensing, counting, and containment. Cell Rep. 30, 3139–3148 (2020).<br>
                  [5] Crack, J. C. et al. Signal perception by FNR: the role of the iron-sulfur cluster. Biochem. Soc. Trans. 36, 1144–1148 (2008).<br>
                  [6] Alon, U. An Introduction to Systems Biology: Design Principles of Biological Circuits 3–19 (Chapman & Hall/CRC, 2007).<br>
                  [7] Michaelis–Menten Equation. In: Encyclopedia of Genetics, Genomics, Proteomics and Informatics. Springer, Dordrecht. https://doi.org/10.1007/978-1-4020-6754-9_10317 (2008).<br>
                  [8]Gang, X. U. , H. Wen , and W. U. Kun . "Primal chaos data system and feedback control research of Logistic population increase model." Journal of Natural Science of Heilongjiang University (2003).<br>
                  [9] Mohammad Reza Bonyadi, Zbigniew Michalewicz; Particle Swarm Optimization for Single Objective Continuous Space Problems: A Review. Evol Comput 2017; 25 (1): 1–54. doi: https://doi.org/10.1162/EVCO_r_00180
                  </p>
              </div>           
    </div>
  </div>
</div>


{% endblock %}
